lib/bundle.js

```diff
@@ -76,6 +77,101 @@ function extractBundleName(fileName) {
   return path.relative(config.pjson.baseURL, fileName.replace(/\.js$/, '')).replace(/\\/g, '/');
 }
 
+function getModuleName (file) {
```

Comment:
@guybedford Great! that makes the `path sorter function` obsolete too which you mentioned in the other comment.